tools/biobb_pytorch/biobb_train_mdae.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="biobb_pytorch_train_mdae" name="TrainMdae" version="@TOOL_VERSION@" profile="22.05">
+    <description>Train a Molecular Dynamics AutoEncoder (MDAE) PyTorch model.</description>
+    <macros>
+      <token name="@TOOL_VERSION@">4.2.1</token>
+    </macros>
+
+    <requirements>
+      <requirement type="package" version="@TOOL_VERSION@">biobb_pytorch</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+
+      ln -s '$input_train_npy_path' ./input_train_npy_path.$input_train_npy_path.ext &&
+
+      #if $input_model_pth_path:
+        ln -s '$input_model_pth_path' ./input_model_pth_path.$input_model_pth_path.ext &&
+      #end if
+      #if $config_json:
+        ln -s '$config_json' ./config_json.$config_json.ext &&
+      #end if
+
+      train_mdae
+
+      #if $config_json:
+        --config ./config_json.$config_json.ext
```

Comment:
```suggestion
        --config ./$train_config
```

Reply:
That’s something we’re planning for a future release, where the most relevant properties will be integrated into Galaxy's UI through sliders, multi-select options, number validators, filters, etc., making the configuration process more user-friendly. However, for now, I’d like to keep things as simple as possible and focus on getting my first tool published in the Galaxy Toolshed.